docs/running-workflows/supported-systems/serial/index.md

```diff
@@ -1,3 +1,6 @@
 # Serial
 
-A simple serial or sequential workflow system.
+The Serial workflow system is using the sequence workflow pattern where each
```

Comment:
To better illustrate Serial workflows, what do you think about including a simple DAG, mentioning inputs/outputs?

Something along this line:

```mermaid
stateDiagram-v2
  [*] --> step_1: inputs
  step_1 --> step_2: step_1 outputs
  step_2 --> step_3: step_2 outputs
  step_3 --> [*]: final result
```

Let me know, peharps is overkill.

Reply:
I'm not sure, in one hand we have some attempts to illustrate Serial workflow in the [docs](https://docs.reana.io/running-workflows/what-is-workflow/), but it does not look great IMO. So it would be nice to dedicate more effort into producing nice workflow visualizations similar to the one in your example to all workflow systems, perhaps could be done in a separate issue/PR?

Reply:
OK, perhaps it's enough to keep this info in the ["What is a workflow?"](https://docs.reana.io/running-workflows/what-is-workflow/) page. I had missed that.